app.rb

```diff
@@ -175,6 +175,50 @@ def respond_with(format, category, responses)
     end
   end
 
+  def form_xml_for_settings(tag, values)
+    Nokogiri::XML::Builder.new do |xml|
+      xml.root do
+        values.each { |value| xml.send(tag, name: value) }
+      end
+    end.doc.css(tag)
+  end
+
+  get '/v1/providers.?:ext?' do
+    validate_params_ext
+
+    responses = form_xml_for_settings('provider', settings.providers)
```

Comment:
We should come back with another PR and make this dynamic driven based off the .xml file names in the data location. Then it will be possible to onboard a new provider by simply dropping the data file into the "right" location.

Reply:
:thinking: At the rate we're currently adding providers, I'm happy with maintaining the list here as a validator. If we start adding providers with a frequency that makes the list difficult to maintain, then we can consider that change.